app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java

```diff
@@ -0,0 +1,271 @@
+package com.xa.xpensauditor;
+
+import android.app.Activity;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.firebase.client.Firebase;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.Calendar;
+
+public class SetLimitActivity extends AppCompatActivity {
+    private Button setdailylimit, setmonthlylimit, setannuallimit, savebutton, clearbutton, retrievebutton;
+    private ProgressBar progressBar;
+    private TextView daylimitAmt, monlimitAmt, annlimitAmt;
+    private FirebaseAuth auth;
+    private Firebase mRootRef;
+    private Firebase RefUid;
+    Activity activity;
+
+    public static final String SHARED_PREFS = "MySharedPref";
+    public static final String DLIMIT = "DAILY_LIMIT";
+    public static final String MLIMIT = "MON_LIMIT";
+    public static final String ALIMIT = "ANN_LIMIT";
+
+    String day, month, year;
+    private String Uid, mail;
+    private int dt, mt, yt;
+
+
+//    protected void AnnLimitEmailsender(String yr, String email) {
+//        Toast.makeText(activity, "TODO - SEND EMAIL TO THE USER AS YEARLY SET LIMIT IS EXCEEDED ", Toast.LENGTH_SHORT).show();
+//        new LimitEmailSender(yr, email).execute();
+//        return;
+//    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 52.